src/common/database/commands/save/RemoveRelationCommand.ts

```diff
@@ -56,7 +56,7 @@ export class RemoveRelationCommand extends DatabaseCommand<void> {
      * @param secundary the secundary id column name
      */
     public static fromPrimary(primary: string, secundary: string): (id: string, node: CCIMSNode) => RemoveRelationCommand {
-        return (id, node) => new RemoveRelationCommand(`relation_${primary}, ${secundary}`, primary + "_id", secundary + "_id", node.id, id);
+        return (id, node) => new RemoveRelationCommand(`relation_${primary}_${secundary}`, primary + "_id", secundary + "_id", node.id, id);
```

Comment:
LOL, how did that even work before
And why haven't we found that earlier?

Reply:
it never did
because we never tested 😂

Reply:
Oopsie 🙃